service/history/hsm/tree.go

```diff
@@ -170,6 +196,7 @@ func (n *Node) Dirty() bool {
 type TransitionOutputWithCount struct {
 	TransitionOutput
 	TransitionCount int64
+	NodePath        []Key
```

Comment:
This is redundant, `PathsAndOutputs` already contains a path. No need to duplicate this info.

Reply:
Fixed